data/index.json

```diff
@@ -0,0 +1,177 @@
+        {
+          "name": "ap-sg-1",
+          "url": "http://localhost"
+        },
+        {
+          "name": "ap-sg-2",
+          "url": "http://localhost"
+        },
+        {
+          "name": "ap-sg-3",
+          "url": "http://localhost"
+        }
+      ]
+    },
+    {
+      "name": "Seoul, South Korea",
+      "continent": "ASIA",
+      "datacenters": [
+        {
+          "name": "ap-kr-1",
+          "url": "http://localhost"
+        },
+        {
+          "name": "ap-kr-2",
+          "url": "http://localhost"
+        },
+        {
+          "name": "ap-kr-3",
+          "url": "http://localhost"
+        }
+      ]
+    }
+  ],
+  "categories": [
+    {
+      "name": "Compute",
+      "services": [
+        {
+          "name": "VMs & Containers",
+          "slug": "instances",
+          "description": "Run VMs and bare metal containers"
+        }
+      ]
+    },
+    {
+      "name": "Network",
+      "services": [
+        {
+          "name": "VLANs",
+          "slug": "vlans",
+          "description": "Wire your application your way"
+        },
+        {
+          "name": "Subnets",
+          "slug": "subnets",
+          "description": "A network for everything"
+        },
+        {
+          "name": "Firewall Rules",
+          "slug": "firewall",
+          "description": "Control the bits coming and going"
+        }
+      ]
+    },
+    {
+      "name": "Storage",
+      "services": [
+        {
+          "name": "Triton Object Storage",
+          "slug": "object-storage",
+          "description": "Modern cloud object storage",
+          "tags": ["'note'='was Manta'"]
+        },
+        {
+          "name": "S3 Compatibility Bridge",
+          "slug": "s3-bridge",
+          "description": "Modern storage, legacy compability"
+        },
+        {
+          "name": "Triton Volumes",
+          "slug": "volumes",
+          "description": "Network filesystems for your apps",
+          "tags": ["'is-new'='true'"]
+        }
+      ]
+    },
+    {
+      "name": "Access",
+      "services": [
+        {
+          "name": "Role Based Access Control",
+          "slug": "rbac",
+          "description": "Manage users within your account"
+        }
+      ]
+    },
+    {
+      "name": "Help & Support",
+      "services": [
+        {
+          "name": "Service Status",
+          "slug": "status",
+          "description": "Find out about the status of our services"
+        },
+        {
+          "name": "Contact Support",
+          "slug": "contact-support",
+          "description": "Chat to us via phone or email"
+        },
+        {
+          "name": "Support Plans",
+          "slug": "support-plans",
+          "description": "Write here about Support Plans"
+        },
+        {
+          "name": "Getting Started",
+          "slug": "getting-started",
+          "description": "Write here about Getting Started"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
Looks like EOL issue here.  Probably a setting in your editor.  I'll let Wyatt comment on the rest but it will impact our current deploys.